README.md

```diff
@@ -16,6 +16,7 @@ Currently, this toolkit has the following commands:
 | [repadolog](https://worldbank.github.io/repkit/reference/repadolog.html) | Outputs a report of what commands are installed in the PLUS folder. |
 | [repkit](https://worldbank.github.io/repkit/reference/repkit.html) | Command named the same as the package. Most important purpose is that this command makes the code `which repkit` work. |
 | [reproot](https://worldbank.github.io/repkit/reference/reproot.html) | This command allows teams to dynamically set root-paths with no manual user-specific set-up, in both single-rooted and multi-rooted projects. |
+| [reproot_setup](https://worldbank.github.io/repkit/reference/reproot_setup.html) | This command helps setting up the environment setting file used in `reproot` |
```

Comment:
Does this even need to be surfaced to users? I added an auto-start from the main command, that may be enough!

Reply:
No strong preference. But in the future, I want `reproot_setup` to also be a tool where the content of the environment file is managed through a GUI window in Stata. Then this needs to exists and have a helpfile etc. Until then this could be an undocumented command.

My weak preference to keep it is that if people struggle to use it, it would be good to have this to point to. What do you think?

Reply:
I think for that purpose it might be easier to have a subcommand in `reproot` to call it instead of documenting it separately?

Reply:
The options are too different in `reproot` and `reproot_setup`. I think it might make sense as it is now, but not where I want to take this command. And since backward compatibility is a pain in Stata I think it is best to do the long term solution from the beginning.

Reply:
ok! in that case I would just write shortcuts in to report to launch the setup in addition, as I did in this case where no env file is detected

Reply:
Oh, yes. I agree with your contribution to have `reproot` run `reproot_setup` if no env file is found. I never intended to remove that. So lets keep  `reproot_setup` in the web docs _and_ keep the shortcut to  `reproot_setup` in  `reproot`. I.e. leave those things as is in this PR.